pycln.toml

```diff
@@ -1,23 +1,23 @@
 [tool.pycln]
 # Many imports with side-effects
-all = false
+all = true
 skip_imports = [
-  # Star imported C wrapped modules
-  "_winxptheme",
-  "dde",
-  "ntsecuritycon",
-  "odbc",
+  # TODO: Confirm whether these produce side-effects,
+  # if yes: add to the list below
+  # if not: remove from here
+  "pythoncom",
+  "pywintypes",
   "win32api",
-  "win32clipboard",
-  "win32com",
-  "win32cryptcon",
-  "win32event",
   "win32file",
-  "win32gui_struct",
   "win32gui",
-  "win32inet",
-  "win32inetcon",
-  "win32pipe",
-  "win32ras",
-  "win32ui",
+  "win32process",
+  "win32security",
+  "win32uiole",
```

Comment:
Do you know if any of these produce side-effects in a way one might want to import the module w/o using any of its symbols?

Reply:
pywintypes might for the loader, but I don't think the rest do.

Reply:
Noted. I'd like to merge this PR as-is w/o tackling additional changes in Python files, so I added it to the comment.